integration/build/build_squash_test.go

```diff
@@ -86,6 +88,8 @@ func TestBuildSquashParent(t *testing.T) {
 		container.WithImage(name),
 		container.WithCmd("/bin/sh", "-c", "cat /hello"),
 	)
+
+	poll.WaitOn(t, container.IsStopped(ctx, client, cid), poll.WithDelay(100*time.Millisecond))
```

Comment:
Looks like the default is 100 miliseconds, so we can remove this as well;

https://github.com/moby/moby/blob/d8fa0581eeeb1caa5d7f2bef4ac58c0366f92a11/vendor/gotest.tools/v3/poll/poll.go#L115

https://github.com/moby/moby/blob/d8fa0581eeeb1caa5d7f2bef4ac58c0366f92a11/vendor/gotest.tools/v3/poll/poll.go#L39-L40